src/WebJobs.Script/Diagnostics/MetricEventNames.cs

```diff
@@ -55,6 +55,9 @@ public static class MetricEventNames
         public const string FunctionInvokeSucceeded = "function.invoke.succeeded";
         public const string FunctionInvokeFailed = "function.invoke.failed";
 
+        // Http worker events
+        public const string HttpWorker = "function.worker.httpworkerpresent";
```

Comment:
metric name starting with `function` are meant to be for function invocations or other function level metrics right? Rename to `hostjsonfileconfigurationsource.httpWorker` ?